### DISCUSSION TITLE
How to set the address of N3 interface in gnb.yaml

### DISCUSSION THREAD

**Discussion by s5uishida** [srsran/srsRAN_Project#249] [General Help] [answered]
Hi,

Is there a way to set the address of the N3 interface in `gnb.yaml`?

In `enb.conf` of srsRAN_4G there is a parameter `gtp_bind_addr`, but I could not find a parameter to set the address of the N3 interface in `gnb.yaml` of srsRAN_Project.

Sorry if there are any misunderstandings or oversights.

**Comment by brendan-mcauliffe** [comment, marked as answer]
Hi @s5uishida, currently the N2 and N3 addresses are set via the same parameter in the config file, `addr`. We do not currently support independent configuration of the N2 and N3 interfaces. This must be mirrored in the core by setting both the AMF and UPF addresses to the address used in the gNB config.

**Reply by s5uishida** [reply]
Hi @brendan-mcauliffe 

Thank you very much for your reply.
I understand the situation.
